File: TennisWebUI/src/app/Player.ts
export interface Player {
    userId: number;
    firstName: string;
    lastName: string;
    email: string;
    passwordHash: string;
    role: string;
    createdAt: string;
    dateOfBirth: string; // Optional field
    level: string; // Player's skill level
    profileImage: string; // URL or path to the profile image
  }

File: TennisASPApp/Models/Player.cs
using System.ComponentModel.DataAnnotations;

namespace TennisAPI.Models
{
    public class Player
    {
        [Key]
        public int UserId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string? PasswordHash { get; set; }
        public string Role { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime? DateOfBirth { get; set; } // Nullable for optional field
        public string? Level { get; set; } // e.g., Beginner, Intermediate, Advanced
        public string? ProfileImage { get; set; } // URL or path to image
    }
}
